package.json

```diff
@@ -39,7 +40,11 @@
     "eslint-config-next": "13.5.6",
     "eslint-config-prettier": "^9.0.0",
     "prettier": "^3.0.3",
-    "typescript": "^5.0.4"
+    "typescript": "^5.0.4",
+    "@types/react-dom": "^18",
+    "autoprefixer": "^10",
+    "postcss": "^8",
+    "tailwindcss": "^3"
```

Comment:
Yay more packages🎉
I understand nextui is now built on tailwind so I guess it has to be

Reply:
Hahah yeah, would've preferred to avoid it - but if we want to use nextui there doesn't seem to be any way around it..

For simplicity it miight be simpler to just scrap nextui some time soon 🤷